public/app/app.js

```
@@ -486,194 +486,176 @@ function _presetDefaults(overrides = {}) {
+    description: 'Band-limit and isolate speech for phone / radio recovery',
     gateThresh: -50, gateRange: -60, gateAttack: 5, gateRelease: 200, gateHold: 50, gateLookahead: 5,
-    nrAmount: 80, nrSensitivity: 70, nrSpectralSub: 65, nrFloor: -72, nrSmoothing: 75,
-    eqSub: -12, eqBass: -8, eqWarmth: -4, eqBody: 0, eqLowMid: 2, eqMid: 1, eqPresence: 0, eqClarity: -4, eqAir: -8, eqBrill: -12,
+    nrAmount: 80, nrSensitivity: 68, nrSpectralSub: 62, nrFloor: -72, nrSmoothing: 72,
+    eqSub: -12, eqBass: -8, eqWarmth: -4, eqBody: 0, eqLowMid: 2, eqMid: 1, eqPresence: 0, eqClarity: -2, eqAir: -6, eqBrill: -10,
     compThresh: -20, compRatio: 5, compAttack: 8, compRelease: 120, compKnee: 4, compMakeup: 4, limThresh: -1, limRelease: 40,
-    hpFreq: 300, hpQ: 1.2, lpFreq: 4000, lpQ: 1.0, deEssFreq: 3000, deEssAmt: 8, specTilt: -1, formantShift: 0,
-    derevAmt: 15, derevDecay: 30, harmRecov: 0, harmOrder: 3, stereoWidth: 0, phaseCorr: 0,
-    voiceIso: 85, bgSuppress: 70, voiceFocusLo: 300, voiceFocusHi: 3400, crosstalkCancel: 20,
+    hpFreq: 300, hpQ: 1.2, lpFreq: 4000, lpQ: 1.0, deEssFreq: 3000, deEssAmt: 6, specTilt: -0.5, formantShift: 0,
+    derevAmt: 15, derevDecay: 30, harmRecov: 20, harmOrder: 4, stereoWidth: 0, phaseCorr: 0,
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)